client/src/main/java/org/schlunzis/kurtama/client/fx/controller/MessageShowingController.java

```diff
@@ -0,0 +1,11 @@
+package org.schlunzis.kurtama.client.fx.controller;
+
+import org.schlunzis.kurtama.client.fx.scene.events.SceneChangeMessage;
+
+import java.util.List;
+
+public interface MessageShowingController {
+
+    void showMessages(List<SceneChangeMessage> messages);
```

Comment:
Kleine Frage: Warum `List` und nicht `Collection`?
Soweit ich das sehe, wir ja nur ein `addAll(...)` genutzt, um die hinzuzufügen, da reicht in der Theorie eine `Collection`.

Reply:
Mit der `List` kann die Reihenfolge der Nachrichten bestimmt werden. Mit `Collection` geht das auch, aber nicht so transparent.